dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -346,7 +357,10 @@ def numba_setup_out_shape(out_shape, in_shape):
             "normalized_shape": True,
         },
     ),
-    (fn.fast_resize_crop_mirror, {"crop": [5, 5], "resize_shorter": 10, "devices": ["cpu"]}),
+    (
+        fn.fast_resize_crop_mirror,
+        {"crop": [5, 5], "resize_shorter": 10, "devices": ["cpu"]},
+    ),
```

Comment:
Nitpick: that's not what our formatting tools produce - we have some config for `black`, extending the line length to 100 characters. Please revert the formatting changes and re-run black with the proper config (I think we have black configuration in our repo).